Changes

```diff
@@ -206,6 +206,9 @@ Working version
   style that the compiler driver uses.
   (David Allsopp, review by Gabriel Scherer)
 
+- #12811: Define and use the CAMLthread_local macro for TLS variables.
+  (Antonin Décimo and Samuel Hym, review by Miod Vallat)
```

Comment:
(this would benefit from an update before merging)

Reply:
I've added Xavier Leroy as a reviewer, thanks.